### DISCUSSION TITLE
precommit1 websock connecttionclosed

### DISCUSSION THREAD

**Discussion by 2019jack** [filecoin-project/lotus#4126] [General Q&A] [answered]
2020-10-01 17:34:28 +0800 CST:  [event;sealing.SectorSealPreCommit1Failed]      {"User":{}}
        seal pre commit(1) failed: handler: websocket connection closed
20.     2020-10-01 17:35:28 +0800 CST:  [event;sealing.SectorRetrySealPreCommit1]       {"User":{}}

Daemon:  0.8.1+git.60d442e3.dirty+api0.16.0
Local: lotus version 0.8.1+git.60d442e3.dirty

**Comment by SgtCoin** [comment, marked as answer]
@2019jack Please let me know if you are still having issues. Given that we are on Lotus version 1.13+, I am going to archive this discussion.
